src/test/e2e/28_temp_directory_deploy_test.go

```diff
@@ -0,0 +1,53 @@
+package test
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/internal/k8s"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTempDirectoryDeploy(t *testing.T) {
+	t.Log("E2E: Temporary directory deploy")
+
+	// run `zarf package deploy` with a specified tmp location
+	otherTmpPath := "/tmp/othertmp"
+
+	e2e.cleanFiles(otherTmpPath)
+
+	e2e.setup(t)
+	defer e2e.teardown(t)
+
+	path := fmt.Sprintf("build/zarf-package-dos-games-%s.tar.zst", e2e.arch)
```

Comment:
I'd love to just move this test up to before the `20` init test because we can test against any of those packages that don't need a cluster.  The existing of a cluster is irrelevant for this test.

Reply:
Ok yeah was debating how "thorough" to be but that makes more sense.  I'll go ahead and fix this.